sim.yml

```diff
@@ -0,0 +1,68 @@
+version: "3.3"
+services:
+
+  mavlink2rest:
+    image: ericjohnson97/iq_sim_mavlink2rest:t0.11.19
+    network_mode: host
```

Comment:
Missing double quotes here, not sure if it's necessary, just to be consistent with the others definitions

Reply:
true. It is good to be consistent. I went ahead and changed all the strings to have quotes around them.